man/pdfpc.in

```diff
@@ -124,6 +124,10 @@ Go to the last slide
 .B Backspace
 Go back in history. Note that history is defined by "jump" commands, not by normal slide movement.
 .TP
+.B j / k
```

Comment:
I find it a little counter intuitive that `j` do the next slide and `k` the prev. I would swap both keys